sdcm/scylla_bench_thread.py

```diff
@@ -179,69 +179,63 @@ def _parse_bench_summary(lines):  # pylint: disable=too-many-branches
-                    line.startswith('Maximum rate:') or line.startswith('Client compression:'):
-                # Concurrency:             7
-                # Connections:             4
-                # Maximum rate:            32000 op/s
-                # Client compression:      true
-                split = line.split(':', maxsplit=1)
-                key = split[0].strip()
-                value = split[1].split()[0] if split[1] else ''
-                results[key] = value
-            elif line.startswith('Clustering row size:'):
-                # Clustering row size:     Fixed(5120)
-                split = line.split(':', maxsplit=1)
-                key = split[0].strip()
-                value = ' '.join(split[1].split())
-                results[key] = value
-
             if line.startswith('Results'):
-                enable_parse = True
-                continue
-            if line.startswith('Latency:') or ':' not in line or 'EOF' in line:
                 continue
-            if not enable_parse:
+            if 'c-o fixed latency' in line:
+                # Ignore C-O Fixed latencies
+                #
+                # c-o fixed latency :
+                #   max:        5.668863ms
+                #   99.9th:	    5.537791ms
+                #   99th:       3.440639ms
+                #   95th:       3.342335ms
+                break
+
+            split = line.split(':', maxsplit=1)
+            if len(split) < 2:
                 continue
-
-            split_idx = line.index(':')
-            key = line[:split_idx].strip()
-            value = line[split_idx + 1:].split()[0]
-
-            # the value may be in milliseconds(ms) or microseconds(string containing non-ascii character)
-            value = convert_metric_to_ms(value)
-
-            # we try to use the same stats as we have in cassandra
-            if key == 'max':
-                key = 'latency max'
-            elif key == '99.9th':
-                key = 'latency 99.9th percentile'
-            elif key == '99th':
-                key = 'latency 99th percentile'
-            elif key == '95th':
-                key = 'latency 95th percentile'
-            elif key == 'median':
-                key = 'latency median'
-            elif key == 'mean':
-                key = 'latency mean'
-            elif key == 'Operations/s':
-                key = 'op rate'
-            elif key == 'Rows/s':
-                key = 'row rate'
-                results['partition rate'] = value
-            elif key == 'Total ops':  # ==Total rows?
-                key = 'Total partitions'
-            elif key == 'Time (avg)':
-                key = 'Total operation time'
-            results[key] = value
+            key = split[0].strip()
+            value = ' '.join(split[1].split())
+            if target_key := interested_keys.get(key, None):
+                if target_key is True:
+                    target_key = key
+                if value.isdecimal():
```

Comment:
How it will parse "Client compression: true" ?

Reply:
it would go convert_metric_to_ms and resulted in 'true'